module/workflow/src/Domain/Query/TransitionConditionSetQueryInterface.php

```diff
@@ -0,0 +1,20 @@
+<?php
+/**
+ * Copyright © Ergonode Sp. z o.o. All rights reserved.
+ * See LICENSE.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Ergonode\Workflow\Domain\Query;
+
+use Ergonode\SharedKernel\Domain\Aggregate\ConditionSetId;
+use Ergonode\SharedKernel\Domain\Aggregate\TransitionId;
+
+interface TransitionConditionSetQueryInterface extends TransitionQueryInterface
```

Comment:
It doesn't seem needed to extend the interface to me. Separate interfaces would work just fine